rust/src/builder.rs

```diff
@@ -365,7 +388,7 @@ lazy_static::lazy_static! {
 /// Extra slashes will be removed from the end path as well.
 ///
 /// Will return an error if the location is not valid. For example,
-pub(crate) fn ensure_table_uri(table_uri: impl AsRef<str>) -> DeltaResult<Url> {
+pub fn ensure_table_uri(table_uri: impl AsRef<str>) -> DeltaResult<Url> {
```

Comment:
This I figured might not be a good idea, but is there some other way of using the function in the test?

Reply:
To keep it private, we can keep the tests within this file, in the tests section at the bottom.

Reply:
Hmm, then I have some trouble with the `mod fs_common` where I put the SlowStore implementation. And the tests are not really testing builder per se. Would it make sense to just make a copy of this function in `fs_common` for the purposes of the test?